docs/rules/rrd/script-length.md

```diff
@@ -8,3 +8,48 @@ Checks if the script section of a Vue component is too long. The default thresho
 - **Readability**: Shorter scripts are easier to read and understand. When a script is over 50 lines it does not fit on a single screen, which makes it harder to read. When it is over 100 lines you have to scroll to read it.
 - **Maintainability**: Shorter scripts are easier to maintain and refactor.
 - **Testability**: Shorter scripts are easier to test.
+
+## 😱 Examples of code for which this rule will throw a warning
+
+::: warning
+The following code contains a `<script>` block that exceeds the recommended maximum length of lines, making it difficult to maintain and understand. Long scripts are often a sign that the logic could be better organized or broken down into smaller, more manageable pieces.
+:::
+
+```vue
```

Comment:
In real life is much more about the component doing too much and that is why it is long.
The solution is to break it to sub components.

Reply:
updated in https://github.com/rrd108/vue-mess-detector/pull/133/commits/968724b70b5832653e07fb820e899407bf5384c8